meson_cpu/meson.build

```diff
@@ -207,8 +226,17 @@ foreach opt_name, conf : parse_options
       endforeach
     else
       filterd = []
+      # filter out the features that are in the accumulate list 
+      # including any successor features
```

Comment:
Not excluding successor features should be considered a bug and needs to be back-ported.
For example, with cpu-dispatch before this patch, to exclude AVX512 you had to exclude all successor features, otherwise it would not be disabled:
```
python -m build --wheel -Csetup-args=-Dcpu-dispatch="max -avx512f -avx512cd \
-avx512_knl -avx512_knm -avx512_skx -avx512_clx -avx512_cnl -avx512_icl -avx512_spr"
 ```
After this fix:
```
python -m build --wheel -Csetup-args=-Dcpu-dispatch="max -avx512f"
```